support/list_branches.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# List all branches that generated documentation should target
+
+set -Eeuo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# The root of the OSCAL-Reference repository
+ROOT_DIR="${SCRIPT_DIR}/.."
+OSCAL_DIR="${ROOT_DIR}/support/OSCAL"
+
+[[ -z "${1-}" ]] && { echo "Error: Necessary git remote not set"; usage; exit 1; }
+[[ -z "${2-}" ]] && { echo "Error: Necessary git branch prefix not set"; usage; exit 1; }
```

Comment:
lines must be moved **below** the definition of `usage()`

Reply:
Fixed in pairing.